cadquery/occ_impl/geom.py

```diff
@@ -361,6 +361,35 @@ def __getitem__(self, rc: Tuple[int, int]) -> float:
         else:
             raise IndexError("Out of bounds access into 4x4 matrix: {!r}".format(rc))
 
+    def __repr__(self) -> str:
```

Comment:
How about suing f strings and reusing the transposed? Something along those lines:
```python
l = cq.Matrix().transposed_list()
rv=(f'Matrix({l[0::4]},\n'
    f'       {l[1::4]},\n'
    f'       {l[2::4]},\n'
    f'       {l[3::4]})')
```

Reply:
The transposed_list() method does generate a list but it also transposes the matrix which would have to be undone. To address the comments I've opened another pull request with this code: 
        list = [ [self.__getitem__((r,c)) for c in range(4)] for r in range(4)]
        return "Matrix("+"".join(str(list)).replace("],","],\n       ")+")"
which generates a list much like transposed_list() does and then uses join and some string replacement to generate the same strings as before. The code is more compact but I'll let you decide if it's better.

I also improved the unit test as I realized that using the identity matrix hides accidental transposition of the matrix.

Reply:
Does [0::4] not result in selecting a column (a thus a reverse transpose when printing it in every row)?

Reply:
My comment about the use of the transposed_list() method applied to my solution where I do a str(list) which would not compensate for a transposed matrix. I stand behind my pull request for this trivial change.

Reply:
OK enough bike-shedding, just please make it more readable (i.e. get rid of `__getitem__`)  and fix the black failure

Reply:
I changed out `__getitem__` for `transposed_list`.  All of the checks have passed with this latest revision and there are no conflicts.